app/models/user_history.rb

```diff
@@ -223,7 +223,18 @@ def self.staff_actions
       update_public_sidebar_section
       destroy_public_sidebar_section
       reset_bounce_score
+      update_directory_columns
+      deleted_unused_tags
+      renamed_tag
+      deleted_tag
+      chat_channel_status_change
+      chat_auto_remove_membership
     ]
+
+    actions_constants = [Chat::TrashChannel::DELETE_CHANNEL_LOG_KEY.to_sym]
```

Comment:
Oh careful, chat is a plugin (even though it ships with core). I'm not sure, but this may have side effects (if chat is disabled, for example). 

I would keep this simple for now, and just add the core values to `staff_actions`. We may need to introduce an API here so that plugins can register their staff actions properly here. (But that should be a separate PR.)